nixos/modules/services/networking/dae.nix

```diff
@@ -129,6 +129,8 @@ in
             });
         in
         {
+          after = [ "network-online.target" "systemd-sysctl.service" ];
+          wants = [ "network-online.target" ];
```

Comment:
```suggestion
```
Network should come up automatically on `nixOS` and doesn't need to be explicitly added to `wants`